rancher/distros-test-framework
# Cluster Reset Restore Path

#### Proposed Changes ####

Adding cluster reset restore path test to the automation framework





#### Types of Changes ####

clusterrestore added to framework for release testing


Patch Validation

#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.


2. Verify code lint; we should not have errors.


3. Update the documentation if needed.
Yes

4. Update makefile and docker run if adding new tests.


5. Run your tests at least 4 times with all configurations needed and possible.


6. If needed test with different os types.
Yes

#### Linked Issues ####



#### Further Comments ####
